core/src/test/scala/unit/kafka/controller/ControllerChannelManagerTest.scala

```diff
@@ -873,6 +883,11 @@ class ControllerChannelManagerTest {
       context.updatePartitionFullReplicaAssignment(partition, ReplicaAssignment(replicas))
       leaderIndex += 1
     }
+
+    context.allTopics ++= topics
+    for (topic <- topics if topicIds.contains(topic)) {
```

Comment:
Couldn't we just do this instead of the for loop:
```
 topicIds.foreach { case (name, id) => context.addTopicId(name, id) }
```

Reply:
done!